mkdocs.yml

```diff
@@ -266,9 +275,9 @@ nav:
             - 'Backchannel Authentication': 'admin/auth-server/endpoints/backchannel-authentication.md'
         - 'Crypto':
             - 'admin/auth-server/crypto/README.md'
-            - 'Supported Algorithms': 'admin/auth-server/crypto/supported-algorithms.md'
+            - 'Keys': 'admin/auth-server/crypto/supported-algorithms.md'
```

Comment:
Name of the file can also change so that the term `keys` is consistently used. `supported-algorithms.md` -> `supported-key-algorithms.md`

Reply:
updated to `keys.md`